### DISCUSSION TITLE
How to get/download configured/updated values.yaml file of deployed application in ArgoCD dashbaord?

### DISCUSSION THREAD

**Discussion by VeereshPatil** [argoproj/argo-cd#11406] [Q&A]
I have deployed an application in ArgoCD using open source helm charts(created application using using argo UI dashboard). I have modified/added some parameters in ArgoCD UI.

I need the values.yaml file with updated data so that I can use it to deploy locally or anywhere using helm command.

The one way is to see the ArgoCD UI, and copy each parameter manually which will take lot of time since I have lot of parameters set.

Is there any easy way to download the values.yaml in argo UI or using argo CLI?

**Comment by kostis-codefresh** [comment]
If you get the application with `kubectl get apps` you should be able to get the full manifest of the application as ArgoCD knows it. Did you try this?

In general, creating applications from the Argo CD UI is only good for experimentation and examples. Production use should always use files in Git (the whole GitOps concept).

**Reply by VeereshPatil** [reply]
Thanks for the response.
"kubectl get apps" is not showing the apps created and deployed using Argo UI.
